### DISCUSSION TITLE
I need access to GitHub Codespaces

### DISCUSSION THREAD

**Discussion by nathanzebedee** [community/community#32672] [Codespaces]
Hello, I am trying to enable GitHub Codespaces for my organization, and GitHub.com says to contact the GitHub team? Can someone please enable GutHub codespaces for my account.

**Comment by dmilburn** [comment]
Hi @nathanzebedee, thanks for reaching out!  I wanted to make sure you saw this docs page with instructions on setting up codespaces for your organization: https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/enabling-github-codespaces-for-your-organization

Currently, Codespaces isn't available to individuals, only organizations. From those docs: 

> GitHub Codespaces is available for organizations using GitHub Team or GitHub Enterprise Cloud.

If those don't docs don't help, if you could share the name of your organization, I can take a deeper look.
